sdks/java/core/src/main/java/org/apache/beam/sdk/util/ZipFiles.java

```diff
@@ -226,6 +228,8 @@ static void zipDirectory(
    * @throws IOException the zipping failed, e.g. because the input was not
    *     readable.
    */
+  @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE",
```

Comment:
I'll add more context to the justification. FindBugs is complaining becuase `File.listFiles()` is documented to return `null` in some circumstances, and we don't check the returned value before dereferencing.